packages/nextjs/src/client/browserTracingIntegration.ts

```diff
@@ -12,6 +13,16 @@ export function browserTracingIntegration(
     ...options,
     instrumentNavigation: false,
     instrumentPageLoad: false,
+    onRequestSpanStart(...args) {
+      const [span, { headers }] = args;
+
+      // Next.js prefetch requests have a `next-router-prefetch` header
+      if (headers?.get('next-router-prefetch')) {
+        span?.setAttribute(SEMANTIC_ATTRIBUTE_SENTRY_OP, 'http.client.prefetch');
+      }
```

Comment:
Q: Have we checked if this change plays well with the product? Specifically I'm thinking of the "Network Requests" insights module 
![image](https://github.com/user-attachments/assets/a32f79dc-8041-46c3-8031-c999225b9a5f)


wondering if `http.client.prefetch` spans would still count towards the requests tracked in this module. 

Also, might be worth adding the op to [dev docs](https://develop.sentry.dev/sdk/telemetry/traces/span-operations/), wdyt?

Reply:
Very good point https://github.com/getsentry/sentry-docs/pull/13239

Reply:
Based on an internal discussion we have agreed to instead add a `http.request.prefetch: true` attribute to the requests.